<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QMainWindow" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textClientLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLogIn">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDisconnect">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="certificates">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>141</width>
      <height>151</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <widget class="QWidget" name="tabValid">
     <attribute name="title">
      <string>Valid</string>
     </attribute>
     <widget class="QListWidget" name="listCertValid">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>137</width>
        <height>127</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabInvoked">
     <attribute name="title">
      <string>Invoked</string>
     </attribute>
     <widget class="QListWidget" name="listCertInvoked">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>137</width>
        <height>127</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonGetCertificates">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Certificates</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeCompany">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
